webapp/src/components/modals/attach_comment_to_issue/index.js

```diff
@@ -7,7 +7,7 @@ import {bindActionCreators} from 'redux';
 import {getPost} from 'mattermost-redux/selectors/entities/posts';
 import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
 
-import {closeAttachCommentToIssueModal, attachCommentToIssue} from 'actions';
+import {closeAttachCommentToIssueModal, attachCommentToIssue} from '../../../actions';
```

Comment:
It's my ide, it can't seem to follow the 'action' path when it's a plugin. It has no problem with that path style in the webapp project, probably because it's a regular react project, not a nested react project. 

I can change it back if you're really against it, but I would prefer to change everything to a relative path personally -- it makes navigating code so much easier for me. Otherwise I'm typing the function name everytime I need to trace a function in a plugin.